WordPress/wordpress-playground
# Apply browser-targeted WordPress patches via Blueprints, not in remote.html

Enables patching arbitrary WordPress bundles for compatibility with web
browsers. Without this PR, remote.html applies these patch right after
the WordPress module is loaded into the /wordpress VFS directory.
However, in https://github.com/WordPress/wordpress-playground/pull/700 we want to enable using an arbitrary WordPress build
from any PR in wordpress-develop. We must, thus, expose a way of
applying the necessary patches.

## Testing instructions

1. Start Playground, log out
2. Go to /wp-login.php
3. Confirm there's a message explaining what login credentials to use

<img width="419" alt="CleanShot 2023-10-16 at 12 40 13@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/9d775c67-c9cb-4826-bd90-9ef766c697b2">
